x/auth/account.go

```diff
@@ -60,6 +60,13 @@ type VestingAccount interface {
 	GetDelegatedVesting() sdk.Coins
 }
 
+// ModuleAccount defines an account type for modules that hold tokens as an escrow
+type ModuleAccount interface {
+	Account
+
+	GetModuleName() string
```

Comment:
```suggestion
	ModuleName() string
```

Reply:
Yeah we should avoid `Get*` in interfaces (and types where members are private).